search/run_search_engine.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python2.7
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
+import argparse
+import os
+import search_engine_pylib as search
+
+
+DIR = os.path.dirname(__file__)
+RESOURCE_PATH = os.path.realpath(os.path.join(DIR, '..', 'data'))
+MWM_PATH = os.path.realpath(os.path.join(DIR, '..', 'data'))
+
+parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
```

Comment:
The only purpose of this script is to illustrate how to use search engine interface. It's name is imperative - "run" - therefore it's a bad idea to use it as a module, so I don't see why do I need to implement all these if __name__ == '__main__' stuff.